app/lib/hosting_environment.rb

```diff
@@ -0,0 +1,9 @@
+module HostingEnvironment
+  def self.environment_name
+    ENV.fetch('HOSTING_ENVIRONMENT_NAME', 'unknown-environment')
+  end
+
+  def self.production?
+    environment_name == 'production'
```

Comment:
this way is preferable and we should move towards it, but as Find uses Rails environments for qa, sandbox etc right now, could this be `Rails.env.production?` and lose the ENV var above?